techtv2ovs/utils.py

```diff
@@ -190,6 +195,51 @@ def process_thumbs(self, ttv_video):
             ttv_video.thumbnail_status = ImportStatus.COMPLETE
         ttv_video.save()
 
+    def process_captions(self, ttv_video):
+        """
+        Copy over any caption files for a TechTVVideo from Dropbox to S3.
+        Create a VideoSubtitle object for each.
+
+        Args:
+            ttv_video (TechTVVideo): The TechTVVideo object to process
+        """
+        ttv_video.subtitle_status = ImportStatus.CREATED
+        captions = [caption[0] for caption in mysql_query(
+            'SELECT srt_file_file_name from closed_caption_files where video_id = %s', [ttv_video.ttv_id],
+            self.connection
+        )]
+        for caption in captions:
+            try:
+                basename, _ = splitext(caption)
+                dbx = dropbox.Dropbox(settings.DROPBOX_TOKEN)
+                _, response = dbx.files_download('/ODL-Video/TechTV/Captions/{}'.format(caption))
```

Comment:
Do you know if any of the files in the tech tv set have a slash or other character which needs escaping?

Reply:
No slashes, some have spaces and parentheses but the dropbox module seems to handle those fine without the need to escape.  But I did just notice that the dropbox file names do not consistently match what's in the MySQL database: sometimes the underscores in `closed_caption_file.srt_file_file_name` are replaced with spaces in dropbox, sometimes not.  So I'll modify the code to try it one way, then the other if it fails.